andela/knights-bn-backend
# #170947653 Docker file that will be used to create Docker image and Docker container

#### What does this PR do?
- This PR is adding Dockerfile for creating docker image
#### Description of Task to be completed?
- writing all commands that Docker will use to generate both docker image and docker container
#### How should this be manually tested?
- [clone the repo](https://github.com/andela/knights-bn-backend.git)
- run `npm i` to install the dependencies 
- see if your docker is installed by writing `docker version` if not install it [link](https://docs.docker.com/install/)
- run `docker build -t docker image name .`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947653](https://www.pivotaltracker.com/story/show/170947653)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
